src/main/java/com/keniu/dto/CreateUserRequestDto.java

```diff
@@ -5,15 +5,6 @@
 import jakarta.validation.constraints.Size;
 import lombok.Data;
 
-/**
- * Data Transfer Object (DTO) for
- * representing a user creation request in the books shop application.
- * This class is used to transfer user input data for creating a new user
- * between the client and the server.
- *
- * It includes validation constraints for ensuring valid input values,
- * such as non-blank fields, proper password length, and matching passwords.
- */
```

Comment:
why does your PR contain such a big amount of such changes? Wasn't it possible to merge them first and after it create a PR? It's quite hard to review such a mess :) Let's make it as clean as possible and keep on only changes related to your task

Reply:
Because your colleague asked to remove comments that were present in many files. I intentionally made this separate commit to make it easy to review the changes by commit, without needing to read through multiple files